doc/source/mem_vs_db_usecases.rst

```diff
@@ -0,0 +1,3 @@
+Choosing the Right Storage Mode: Memory vs DB Storage
+-----------------------------------------------------
+
```

Comment:
Is this file supposed to be mostly blank? (If the file is no longer needed we should delete it)